command/node_status.go

```diff
@@ -384,17 +392,26 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 
 }
 
+func (c *NodeStatusCommand) outputTruncatedNodeDriverInfo(node *api.Node) string {
+	drivers := make([]string, 1)
```

Comment:
Make the length of the initialized list equal to `len(node.Drivers`. Also it should be `make([]string, 0, len(node.Drivers))`. You are setting the initial length to 1 and then appending to it so index 0 here is empty